src/test/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectActionTest.java

```diff
@@ -72,7 +72,7 @@ public void showErrorDialog(@NotNull String message, @NotNull String title) {
   private PostponedRunnable ideRestarter;
   private TestDialogs dialogs;
 
-  class DummySettingsImporter implements SettingsImporter {
+  class DummySettingsImporter extends SettingsImporter {
```

Comment:
Yep true :smile: Since `SettingsImporterImpl` was completely stateless I got an overwhelming urge to remove it and move the implementation to `SettingsImporter`